### PR TITLE
feature to fix issue #288

### PR DESCRIPTION
**Pull request by clement-pannetier**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

This feature allows you to choose if you want to display the footer or not.
I added a new config parameter "footer" that takes a boolean (true or false) and added the "if" condition in the partial footer file.

### Issues Resolved

Issue #288 

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [ ] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [ ] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by luizdepra** [comment]
The footer template is getting really messy. It needs a major refactor.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-  <section class="container">
-    {{ with .Site.Params.footercontent | safeHTML }}
-      <p>{{.}}</p>
-    {{ end }}
-    {{ if not .Site.Params.hideCopyright }}
-      {{ with .Site.Params.since }}
-        © {{ if lt . now.Year }}{{ . }} - {{ end }}{{ now.Year }}
-      {{ else }}
-        © {{ now.Year }}
+{{ if .Site.Params.footer }}
```
Comment (luizdepra) [inline layouts/partials/footer.html]:
```suggestion
{{ if .Site.Params.footer | default true }}
```
Footer should be displayed by default.

**Comment by luizdepra** [comment]
We need to start using better and meaningful names for configurations.
How about `showFooter`? Also, add it to the `config.toml` as `true`.

**Comment by clement-pannetier** [comment]
> We need to start using better and meaningful names for configurations.
> How about `showFooter`? Also, add it to the `config.toml` as `true`.

Ok, what about `hideFooter`, it follows the others theme parameters like `hidecredits` or `hidecopyright`. In that respect, why they are not in camel case format ?

What do you mean by "Also, add it to the `config.toml` as `true`.". In the exampleSite directory ?

**Comment by luizdepra** [comment]
> What do you mean by "Also, add it to the config.toml as true.". In the exampleSite directory ?
Yup.

**Comment by clement-pannetier** [comment]
Hey, do you now how to update the wiki, I don't have access to it.
I wanted to add this line : 
`| hideFooter | bool | No | If active, hide the footer. | `true` or `false` |`

**Comment by luizdepra** [comment]
Nice, thx.

Well, only I'm able to modify the wiki. I'll create a issue to update it later.

**Comment by luizdepra** [comment]
Nah, just updated it. But I still need to add all the new configurations.

**Pull request merged**